source/agora/consensus/Validation.d

```diff
@@ -631,6 +631,268 @@ unittest
     }
 }
 
+/// Test for transactions having no input or no output
```

Comment:
Looks like the coverage report is really screwed up tho.
It might come from DMD, and I don't think there's much we can do about it.